src/_content/posts/call-for-2025-volunteers.md

```diff
@@ -0,0 +1,319 @@
+- Write a CoC transparency report to recap reports (heavily anonymized) and actions taken
+
+## Social Events Committee
+
+Size: 2-3
+
+The social events committee is responsible for scheduling the "after hours" events at DCUS to give people chances to socialize and unwind after a long day of talks.
+
+### Pre-conference responsibilities
+
+- Find a venue for the speaker appreciation dinner and coordinate contract signing with DEFNA
+- Plan opening reception (if sponsored, this will become a bigger event compared to previous years)
+- Plan any other fun events like board game night
+
+### At-conferece responsibilities
+
+- Work with venue on any work needed for social events
+
+## Venue Committee
+
+Size: 1-2
+
+The venue committee is responsible for coordinating plans between the other organizing teams and the venue(s).
+
+### Pre-conference responsibilities
+
+- Work with event venue to select and lay out talk, tutorial, and sprint rooms
+- Lay out the extra rooms needed for the conference (lactation room, quiet room, organizers room, speaker green room, lunch/break areas)
+- Work with event venue and DEFNA to select catering menus (and providers if we're not contractually obligated to use one)
+- Work with hotel to coordinate room blocks
+- Work with venue to plan logistics (sponsor gear shipping, signage, etc.)
+- Allocate reward rooms in coordination with DEFNA board
+- Order supplies for restroom stocking
+
+### At-conference responsibilities
+
+- Review room guest list against ticket list to claim reward nights
+- Handle any last-minute requests from vendors/sponsors/organizers/etc.
+- Notify venue staff of any needs (bathroom cleanliness, meals to accommodate special diets, etc.)
+
+## Audio/Video (A/V) Committee
+
+Size: 2-3
+
+The A/V committee is responsible for making everything look good. They handle coordination between the on-site crew, the online streaming platform, and the post-conference processing teams and upload the results to YouTube after the conference.
+
+### Pre-conference responsibilities
+
+- Select on-site video production crew
+- Select post-conference processing crew
+- Coordinate both of the above teams with the online platform
+- Work with the venue team to ensure high-speed wired internet access to the on-site video crew
+- Transfer pre-recorded talks to the post-conference processing crew and then the online conference provider
+- Procure a wide variety of dongles for presenters to hook their laptops up to the projectors
+- Procure projectors and screens if the venue doesn't already have them, including one for the green room so speakers can test their setup
+
+### At-conference responsibilities
+
+- Help on-site crew set up equipment and ensure streaming works
+- Get dongles to the right rooms as needed
+- Help keynote speakers get presentation mode working if needed
+- Answer lots and lots of questions
+
+## Online Team
```

Comment:
@blulady I attempted to define the online responsibilities as best I could. It could be improved though for sure